src/controllers/playback/video/index.js

```diff
@@ -1620,6 +1610,44 @@ export default function (view) {
         }
     }
 
+    function updateTrackButtonsState(currentPlaylist) {
+        try {
+            if (currentPlaylist) {
+                const currentPlaylistIndex = playbackManager.getCurrentPlaylistIndex() + 1;
+
+                if (btnPreviousTrack) {
+                    if (currentPlaylistIndex === 1) {
+                        btnPreviousTrack.classList.add('hide');
+                        btnPreviousTrack.disabled = true;
+                    } else {
+                        btnPreviousTrack.classList.remove('hide');
+                        btnPreviousTrack.disabled = false;
+                    }
+                } else {
+                    console.error('[VideoPlayer] failed to get "btnPreviousTrack"', btnPreviousTrack);
+                }
+
+                if (btnNextTrack) {
+                    if (currentPlaylist.length === currentPlaylistIndex) {
+                        btnNextTrack.classList.add('hide');
+                        btnNextTrack.disabled = true;
+                    } else {
+                        btnNextTrack.classList.remove('hide');
+                        btnNextTrack.disabled = false;
+                    }
+                } else {
+                    console.error('[VideoPlayer] failed to get "btnNextTrack"', btnNextTrack);
+                }
+
+                if (currentPlaylist.length === 1 || currentPlaylistIndex === currentPlaylistIndex) {
+                    showOsd();
+                }
+            }
+        } catch (err) {
+            console.error('[VideoPlayer] failed to get playlist', err);
+        }
```

Comment:
I think you should move `try..catch` to `updatePlaylist` where `getPlaylist` is called.